### ISSUE TITLE
What is low-light training data in `Bi-directional low-level alignment`?

### ISSUE THREAD

**Issue by lyf1212** [ZhangYh994/WiiD#2]
As I understand, in `Bi-directional low-level alignment`, you use adversarial and MMD loss to train a distribution-level mapping from `daytime image` to `enhanced image`.

But I cannot find any instruction about the low-light training data in this part. Can you provide some information?

Also, MMD loss is based on downstream network, is it the pretrained one in daytime? Or the whole system is trained end-to-end?

**Comment by ZhangYh994** [comment]
Hello, thank you for your question.
Your understanding might be slightly off. Actually, we can use any low-light enhancement method (preferably with moderate enhancement rather than enhancing to near-normal light levels to avoid excessive noise introduction, etc.) to process the low-light data. Then, we train a GAN network using adversarial loss to transform real normal-light images into the form of the moderately enhanced low-light images mentioned earlier—that is, pseudo-low-light images.
Subsequently, the MMD loss is trained end-to-end. Essentially, while training with darkened normal-light data, it also passes real low-light data (randomly sampled without requiring labels) through the network for feature extraction. However, these features are only used for MMD loss computation and do not participate in labeled training.
Certainly, we have provided the original dataset and the preprocessed dataset, which includes moderately enhanced low-light data and darkened normal-light data:
https://drive.google.com/drive/folders/1d0rs9w4iVBL9PF370ciAmOogSAaqJVqe?usp=drive_link
For the darkening operation on the normal-light dataset, you can download the model from the following link and use the author’s GAN network to perform the darkening yourself. During training, the source domain is the normal-light dataset, and the target domain is the moderately enhanced low-light data:
https://github.com/taesungp/contrastive-unpaired-translation

**Comment by lyf1212** [comment]
I see. Thanks a lot!
